backend/typescript/services/implementations/adminService.ts

```diff
@@ -112,7 +112,19 @@ class AdminService implements IAdminService {
       );
       throw error;
     }
-    return form;
+    return {
+      formQuestions: form.formQuestions.map((formQuestion: FormQuestionDTO) => {
```

Comment:
why could we not have returned the `form` here?

Reply:
I am guessing it's because of the | null. In other functions we rely on returning in the try block and then the catch block would return an error. Would that work here?

Reply:
I found experimentally that when `form` was returned, the `FormQuestion`s would only have an `_id` and not an `id` associated with them.

Reply:
oh right I missed that. I see that we are using a similar approach in other functions too